apps/builder/app/shared/share-project/share-project.tsx

```diff
@@ -237,11 +276,15 @@ export type LinkOptions = {
   token: string;
   name: string;
   relation: Relation;
+  canCopy: boolean;
+  canClone: boolean;
 };
 
-type SharedLinkItemType = LinkOptions & {
-  onChangeRelation: (permission: Relation) => void;
-  onChangeName: (name: string) => void;
+type SharedLinkItemType = {
+  // onChangeRelation: (permission: Relation) => void;
+  // onChangeName: (name: string) => void;
```

Comment:
Why is it commnted out?

Reply:
Because you wasn't invited for review ;-), not finished yet